### PR TITLE
fix: braavos description, icon, tags

### PR DESCRIPTION
**Pull request by avimak**

### CODE REVIEW

**Comment by 0xlny** [comment]
Hey @avimak 👋 
Can you please fix the syntax following the .eslintrc.json & remove the space char in name ? ;)

**Comment by avimak** [comment]
thanks @0xlny, sorry about that! pushed (my git diff settings are set to ignore whitespace changes)

File: data/ecosystem.ts [hunk 1]
```diff
@@ -859,18 +859,19 @@ export const allProjects: Array<Project> = [
     isTestnetLive: true,
   },
   {
-    name: "Braavos",
+    name: " Braavos ",
```
Comment (janek26) [inline data/ecosystem.ts]: nice try!

Comment (avimak) [reply data/ecosystem.ts]: not sure I get it :) ?

**Review by 0xlny — APPROVED** [review]

**Pull request merged**